Software/Development/SemesterProject/App/closedLoopControlManager.c

```diff
@@ -22,29 +22,28 @@
 #include "closedLoopControlManager.h"
 
 static int systemIsInStartup = 1;
+static int16 maxDuty = 10;

```

Comment:
I think this variable name is a bit misleading, it should be more something like dutyRatio. In fact it gets updated calling function getDutyRatio()